Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1450,6 +1458,17 @@ - (void)applyTheme:(WMFTheme *)theme {
     [self setNeedsStatusBarAppearanceUpdate];
 }
 
+- (void)changeTheme:(NSNotification *)note {
+    NSDictionary* userInfo = note.userInfo;
+    WMFTheme* theme = (WMFTheme*)userInfo[@"theme"];
```

Comment:
It's helpful to define user info key strings as constants as well to avoid typos with the string - in this case, something like `WMFUserDidSelectThemeNotificationThemeKey`